src/Components/InstanceTypesSelect/index.js

```diff
@@ -43,15 +43,17 @@ const InstanceTypesSelect = () => {
       <SelectOption
         aria-label={'Instance Type item'}
         key={instanceType.id}
-        description={
-          instanceType.cores +
-          ' cores | ' +
-          instanceType.vcpus +
-          ' vCPU | ' +
-          (parseFloat(instanceType.memory) / 1024).toFixed(1) +
-          ' GiB memory | ' +
-          instanceType.architecture
-        }
+        description={`${instanceType.cores} cores | 
+          ${instanceType.vcpus} vCPU | 
+          ${(parseFloat(instanceType.memory_mib) / 1024).toFixed(
+            1
+          )} GiB memory | 
+          ${
+            instanceType.storage_gb > 0
+              ? instanceType.storage_gb + ' GB storage | '
+              : 'EBS - only | '
+          }
+          ${instanceType.architecture}`}
```

Comment:
Hehe, in this case templating actually looks worse IMHO, but I don't care much I'm good either way, but seeing it it just looks ugly xD

Reply:
I agree, Should I change it back?

Reply:
yeah... I'd clean it by extracting variables outside to improve readability, something like
```js
const typeStorage =  instanceType.storage_gb > 0
              ? `${instanceType.storage_gb}  GB storage |`
              : 'EBS - only | '

const typeMemory= `${parseFloat(instanceType.memory_mib) / 1024).toFixed(1))} `


description=`${instanceType.cores} cores | 
                       ${instanceType.vcpus} vCPU |
                       ${typeStorage}
                        ${typeMemory} GiB memory |`

```

Reply:
agreed, but I guess we can do that in a follow-up :)
Lets merge so anyone can continue improving :)